src/components/LMap.vue

```diff
@@ -45,61 +52,77 @@ const userIcon = icon({
 });
 
 onBeforeUnmount(() => {
-  if (isNumber(watchId)) {
-    navigator.geolocation.clearWatch(watchId);
-  }
-
   disposeMap();
+  disposeWatcher();
+  disposeMarkers();
```

Comment:
```suggestion
  disposeWatcher();
  disposeMarkers();
  disposeMap();
```

Tenemos que hacerlo en este orden para estar 100% seguros de que no haya condiciones de carrera y se ejecute una actualización del mapa o de los marcadores después de hacer el dispose del mapa, lo que daría problemas.

Reply:
El cambio lo he añadido a este [commit](https://github.com/ispp-2324-ocial/frontend/pull/73/commits/bb1ff4aa3020ecc0c3eb52a502bdaf0822a5dc7c)